src/about.njk

```diff
@@ -113,7 +113,7 @@
         <p class="text-muted">Birmingham leader</p>
         <ul class="list-inline social-buttons">
           <li class="list-inline-item">
-            <a href="https://www.linkedin.com/in/rayrandle/" target="_blank" class="shadow-sm">
+            <a href="https://www.linkedin.com/in/rayrandle/" target="_blank" class="shadow-sm" aria-label="Navigation link to Ray Randle LinkedIn Page">
```

Comment:
Screen readers will announce this as a link, so a better aria-label would be "Ray Randle's LinkedIn Page"